web-client/components/HomeView.js

```diff
@@ -1,14 +1,21 @@
 import { ContentWrapper } from "./ContentWrapper";
-import HomeContentWrapper from "./HomeContentWrapper";
 import Image from "next/image";
 import ViewWrapper from "./ViewWrapper";
+import Button from "./Button";
 
 function HomeContainer() {
     return (
         <ViewWrapper>
             <ContentWrapper>
                 <div className="flex py-24 items-center">
-                    <HomeContentWrapper />
```

Comment:
I removed the HomeContentWrapper because it wasnt that big, and not reused anywhere. If theres additional complexity added to it, we can add it back in

Reply:
cool